src/vsr/grid.zig

```diff
@@ -26,7 +26,7 @@ pub const BlockPtrConst = *align(constants.sector_size) const [constants.block_s
 pub fn allocate_block(
     allocator: mem.Allocator,
 ) error{OutOfMemory}!*align(constants.sector_size) [constants.block_size]u8 {
-    const block = try allocator.alignedAlloc(u8, constants.sector_size, constants.block_size);
+    const block = allocator.alignedAlloc(u8, constants.sector_size, constants.block_size) catch @panic("OOM");
```

Comment:
Not sure what value of `cache-grid` that can be considered to be too unreasonably large to reject during cli parsing. But it seems the effect of a large value is an OOM at this point.